src/redturtle/prenotazioni/restapi/services/booking_info/get.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+from plone import api
+from plone.restapi.interfaces import ISerializeToJson
+from plone.restapi.services import Service
+from zope.component import getMultiAdapter
+from AccessControl import Unauthorized
+
+
+class BookingInfo(Service):
+    def reply(self):
+        catalog = api.portal.get_tool("portal_catalog")
+
+        query = {"portal_type": "Prenotazione", "UID": self.request.uid}
+        booking = catalog.unrestrictedSearchResults(query)
+        booking = booking[0]._unrestrictedGetObject()
```

Comment:
se non ci sono risultati, cosa succede? Si rompe?

Reply:
Come sopra, restituisce 404 (Not found)

Reply:
no, intendo se booking è una lista vuota (non trova nulla la ricerca), facendo booking[0] non da indexerror?

Reply:
come fa a restituire 404? non vedo mica nel codice una return così

Reply:
> come fa a restituire 404? non vedo mica nel codice una return così

ah, visto